README.md

```diff
@@ -1 +1,48 @@
-# capstone-project-module-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project - First Module
+
+> This is my first capstone project building with HTML and CSS.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+If you want to see a live demo of this website, please click on the next link:
+https://dredwell.github.io/capstone-project-module-1/
+
+![Captura de pantalla de 2022-04-23 09-35-04](https://user-images.githubusercontent.com/85886248/164912871-1f42ab5c-9088-426c-beec-c355c7ba3ee3.png)
+
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally. To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Browser (Google Chrome, Mozilla Firefox, Safari, Edge)
+
+### Instalation
+
+Clone the repository or download the files from https://github.com/Dredwell/capstone-project-module-1 and then open index.html in your browser.
+
+## Authors
+
+👤 **Dredwell Velasquez**
+
+- GitHub: [@Dredwell](https://github.com/Dredwell)
+- Twitter: [@dredwell12](https://twitter.com/dredwell12)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dredwell-velasquez-462a60160/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Dredwell/My-Portfolio/issues)
+
```

Comment:
- [ ] Just as the previous reviewer mentioned kindly give credit to `cindy shin` the author of the original design